src/components/Capture/capture.js

```diff
@@ -0,0 +1,239 @@
+import { h, Component } from 'preact'
```

Comment:
any reason for renaming this file? It's convention to name it `index`, since it facilitates the import statement.
If this is a compromise for the IDE, I recommend this:  https://atom.io/packages/smart-tab-name